### ISSUE TITLE
Invite User to AGOL Org Bug

### ISSUE THREAD

**Issue by nfink-nlt** [Esri/arcgis-python-api#491]
ArcGIS Python API v1.6.2

_gis = GIS(portal=org.maps.arcgis.com, username=username, password=password)
gis.users.invite(email='your.name@yourdomain.com')_

returns:
_RuntimeError: Unable to add users.
'emails' parameter required.
(Error Code: 400)_

Attempting to add an "emails" parameter returns:
_TypeError: invite() got an unexpected keyword argument 'emails'_

**Comment by krdyke** [comment]
FYI It looks like the `gis.users.invite` method does not include the `email` param as `emails` when calling `inviteByEmail`. Adding `"emails": email` to the `params` dict in the invite method (line 2148 of gis/_init_.py) seems to fix the issue. Params should look like this:

```python
     params = {
            "f" : "json",
            "emails": email,
            "message" : msg,
            "role" : role,
            "level" : level,
            "targetUserProvider" : provider or "arcgis",
            "mustApprove" : must_approve,
            "expiration" : expiration,
            "validateEmail" : validate_email
        }
```

**Comment by AtmaMani** [comment]
@krdyke we have added this feature and would be available in the upcoming v1.7 of the Python API.
